introduction-to-datasets.md

```diff
@@ -0,0 +1,142 @@
+
+## Partitions
+
+Partitions define optional logical divisions for data storage. For example, you might most often work with data using time-based queries. You can define a partitioning strategy by year, month, and day. When you are using data from January 8, 2015, Hadoop only has to access data stored in the partition `/year=2015/month=1/day=8`. By using partitions that correspond to your most common queries, your applications run more quickly.
+
+Partitioning is optional because there are times when partitioning is not the most efficient solution. However, you should always consider partitioning as a best practice when planning your dataset.
+
+Kite validates your partition strategy using the dataset schema. This catches problems early. For example, if you mistakenly type in your partition definition _timestmap_, validation catches the typo because the schema has a _timestamp_ column. Kite also validates that field types can be partitioned on the selected value. For example, you can't extract a year from a string.
+
+You define your partition strategy in [partition strategy JSON format][ps-format] and apply it when you create your dataset. See [Partitioned Datasets][partition-strategies].
+
+For example, you can use the CLI command [`partition-config`][cli-part-conf] to define a partition strategy using name:value pairs to specify that the dataset should be partitioned on a timestamp (`ts`) field by year, month, and day.
+
+```
+$ kite-dataset partition-config ts:year ts:month ts:day -s rating.avsc
+```
+
+The result is a partition definition in JSON format. 
+
+```JSON
+[ {
+  "name" : "year",
+  "source" : "ts",
+  "type" : "year"
+}, {
+  "name" : "month",
+  "source" : "ts",
+  "type" : "month"
+}, {
+  "name" : "day",
+  "source" : "ts",
+  "type" : "day"
+} ]
+```
+
+If you want to save the partition definition for use with the CLI or API, you can use the `-o` option to output the result to a `.json` file.
+
+[partition-strategies]: {{site.baseurl}}/Partitioned-Datasets.html#partition-strategies
+[cli-part-conf]: {{site.baseurl}}/cli-reference.html#partition-config
+[ps-format]: {{site.baseurl}}/Partition-Strategy-Format.html
+
+## Configuration Options
+
+While you need only a URI and schema to create a dataset, you can use column mapping and properties for additional control over how Kite reads or writes your data.
+
+### Column Mapping
+
+HBase requires that you use column mapping to configure how your records should be stored. You define column mapping in JSON format in a data-centric way. Kite stores and retrieves the data correctly. 
+
+See [Column Mapping][column-mapping]. See also the [CLI mapping-config command][cli-column-mapping].
+
+[column-mapping]: {{site.baseurl}}/Column-Mapping.html
+[cli-column-mapping]: {{site.baseurl}}/cli-reference.html#mapping-config
+
+### Properties
+
+You can add custom settings to dataset properties at creation time for additional control over how Kite reads or writes your data.
```

Comment:
Okay, but I want to say "set" rather than "add" properties, then, to make the distinction.

"You can set properties at creation time for additional control over how Kite reads or writes your data."